repos/fdbt-site/src/pages/selectCapValidity.tsx

```diff
@@ -58,6 +64,7 @@ export const getFieldset = (errors: ErrorInfo[], endOfFareDay?: string): RadioCo
                     id: 'cap-end-of-service-hint',
                     content: "The cap applies to journeys made during the 'fare day' as defined by your business rules",
```

Comment:
These lines of text could be moved to an object, and exported so it can be used on a different page. For example

export const validityHintText = {
endOfCalendarDay: 'The cap applies to journeys made before midnight',
24hr: 'The cap applies to journeys made within 24hrs of the first tap',
fareDayEnd: "The cap applies to journeys made during the 'fare day' as defined by your business rules"
}